src/__test__/details.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Details from '../routers/details';
+
+describe('Details component', () => {
+  it('should render Loading... when weather is null', () => {
+    render(<Details />);
+    expect(screen.getByText('Loading...')).toBeInTheDocument();
+  });
+
+  it('should render the weather data when weather is not null', () => {
+    const mockWeather = {
+      name: 'New York',
+      country: 'US',
+      iconURL: 'https://someicon.com',
+      description: 'sunny',
+      temp: 20,
+      temp_min: 15,
+      temp_max: 25,
```

Comment:
* Some tests are failing, Kindly fix them 
![image](https://user-images.githubusercontent.com/52497246/226113900-a4909439-fd13-499c-81aa-99a702ae9af7.png)

Reply:
- [ ] [REQUIRED] Hello @Chaseknis , there are still failing tests. Let's resolve this.
<img width="339" alt="Screenshot 2023-03-18 at 18 43 10" src="https://user-images.githubusercontent.com/58692590/226124505-21e84fae-252f-4d90-baf2-d9523c078bcd.png">